docs/DEBUGGING.md

```diff
@@ -1,26 +1,60 @@
 ## Xdebug
 
-To use Xdebug in the `frontend` and/or `admin` app, it must be installed on the client image. To install, you will need to either manually pass `REQUIRE_XDEBUG_FRONTEND=1` as an env var when bringing up the app or run the make command `up-app-xdebug-frontend`
+To use Xdebug in the `frontend` and/or `admin` app, it must be installed on the client image. To install, you will need
+to either manually pass `REQUIRE_XDEBUG_FRONTEND=1` as an env var when bringing up the app or run the make
+command `up-app-xdebug-frontend`
 
-To install Xdebug on the API, either manually pass `REQUIRE_XDEBUG_API=1` as an env var when bringing up the app or run the make command `up-app-xdebug-api`.
+To install Xdebug on the API, either manually pass `REQUIRE_XDEBUG_API=1` as an env var when bringing up the app or run
+the make command `up-app-xdebug-api`.
 
 You can confirm installation by running `php -v` in the container and seeing that it reports the Xdebug version.
 
-Since v3 performance with Xdebug enabled is only slightly slower than without so leaving Xdebug enabled shouldn't be a huge performance hit compared with previous versions.
+Since v3 performance with Xdebug enabled is only slightly slower than without so leaving Xdebug enabled shouldn't be a
+huge performance hit compared with previous versions.
```

Comment:
```suggestion
Since v3 performance with Xdebug enabled is only slightly slower than without so leaving Xdebug enabled shouldn't be a huge performance hit compared with previous versions.
```